docs_src/getting-started/Ch-GettingStartedSnapUsers.md

```diff
@@ -64,54 +72,73 @@ $ sudo snap set edgexfoundry support-notifications=off
 All services which are installed on the system as systemd units, which if enabled will automatically start running when the system boots or reboots.
 
 ## Configuring individual services
+The EdgeX snaps support configuration overrides via snap configure hooks which generate service-specific .env files which are used to
+provide a custom environment to the service, overriding the default configuration provided by the service's ```configuration.toml```
+file. If a configuration override is made after a service has already started, then the service must be **restarted** via command-line
+(e.g. ```snap restart edgexfoundry.<service>```), snapd's REST API, or the SMA (sys-mgmt-agent). If the overrides are provided via the
+snap configuration defaults capability of a gadget snap, the overrides will be picked up when the services are first started.
 
-All default configuration files are shipped with the snap inside $SNAP/config, however because $SNAP isn't writable, all of the config files are copied during snap installation (specifically during the install hook, see snap/hooks/install in this repository) to $SNAP_DATA/config.
+The following syntax is used to specify service-specific configuration overrides for the edgexfoundry snap:
 
-```
-$ sudo snap restart edgexfoundry
-```
-## Viewing logs
+```env.<service>.<stanza>.<config option>```
 
-Currently, all log files for the snap's can be found inside $SNAP_COMMON, which is usually /var/snap/edgexfoundry/common. Once all the services are supported as daemons, you can also use sudo snap logs edgexfoundry to view logs.
+For instance, to setup an override of Core Data's Port use:
 
-Additionally, logs can be viewed using the system journal or snap logs. To view the logs for all services in the edgexfoundry snap use:
+```$ sudo snap set edgexfoundry env.core-data.service.port=2112```
 
-```
+And restart the service:
+
+```$ sudo snap restart edgexfoundry.core-data```
+
+**Note** - at this time changes to configuration values in the [Writable] section are not supported.
```

Comment:
Why not?

Reply:
Because the only way to properly support changes to `[Writable]` involves updating Consul directly vs. adding the variable to .env file sourced at service start. While the latter does effectively trigger the override, its not dynamic (i.e. you have to restart the service to pick it up), so we chose not to implement support for `[Writable]` overrides in the first implementation.